crates/net/network/src/eth_requests.rs

```diff
@@ -202,7 +202,11 @@ where
             {
                 let receipt = receipts_by_block
                     .into_iter()
-                    .map(|receipt| receipt.with_bloom())
+                    .map(|receipt| {
+                        // TODO: use `into_with_bloom` on new alloy release
+                        let bloom = receipt.bloom();
+                        ReceiptWithBloom::new(receipt, bloom)
+                    })
```

Comment:
I believe `map(ReceiptWithBloom::from)` should work here

Reply:
oh nice, yep